pom.xml

```diff
@@ -66,6 +66,12 @@
             <artifactId>junit-jupiter-params</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
I don't think Lombok's approved for use in the SDKs, but could you investigate?

Reply:
ideally don't include it - i think we need to add lombok license in this repo before we can do it